helpers/stats/ea_corrplot.m

```diff
@@ -144,7 +154,40 @@
     fs=40;
 end
 %g.set_title([labels{1}, ' [R = ', sprintf('%.2f',R), '; ', pstr, ']'], 'FontSize', fs);
-g.set_title([labels{1}, ' [R = ', sprintf('%.2f',R), '; ', pstr, ']']);
+
+[R_pear,p_pear]=ea_permcorr(X,Y,'pearson');
+pv_pear=p_pear;
+pstr_pear='p';
+
+if pv_pear >= 0.001 % Show p = 0.XXX when p >= 0.001
+    pstr_pear = [pstr_pear, ' = ', sprintf('%.3f',pv_pear)];
+else
+    % pstr = [pstr, ' = ', sprintf('%.1e',pv)]; % Show p = X.Xe-X
+    signCheck=zeros(1,16);
+    for i=1:length(signCheck)
+        signCheck(i)=eval(['pv<1e-',num2str(i),';']);
+    end
+    if all(signCheck)
+        pstr_pear = [pstr_pear, ' < 1e-16']; % Show p < 1e-16
+    else
+        pstr_pear = [pstr_pear, ' < 1e-', num2str(find(diff(signCheck),1))]; % Show p < 1e-X
+    end
+end
+
+
+
+%g.set_title([labels{1}, ' Spearman: [R = ', sprintf('%.2f',R), '; ', pstr, ']']);
+
+
+%if contains(labels{1}, 'TRAIN-TEST') && size(labels, 2) > 4
+if contains(labels{4}, 'nested LOO')
+    g.set_title({['Mean and STD of linear models from nested LOO'],['Slope: ',labels{5}],['Intercept: ',labels{6}]})
+elseif size(labels, 2) > 4    
+    g.set_title({[labels{1}],['Spearman: [r = ', sprintf('%.2f',R), '; ', pstr, ']'],['Pearson: [r = ', sprintf('%.2f',R_pear), '; ', pstr_pear, ']'],[labels{5}, '; ',labels{6}, '; ',labels{7}]});
+else
+    g.set_title({[labels{1}],['Spearman: [r = ', sprintf('%.2f',R), '; ', pstr, ']'],['Pearson: [r = ', sprintf('%.2f',R_pear), '; ', pstr_pear, ']']});
+end
```

Comment:
so now the `p` values shown on the figure are permuted `p` by default??

Reply:
and it was like this